packages/synapse-react-client/src/components/ReleaseCard/ReleaseCardUtils.ts

```diff
@@ -72,16 +80,58 @@ const formatReleaseDate = (
 export const formatReleaseCardData = (
   schema: ReleaseCardSchema,
   data: (string | null)[],
+  releaseMetdataConfig: ReleaseMetadataConfig,
   statsConfig: StatConfig[],
 ) => {
   return {
-    releaseVersion: data[schema.releaseVersion],
-    releaseEntity: data[schema.releaseEntity],
-    releaseDate: formatReleaseDate(schema, data),
+    releaseName: getValueFromData(
+      schema,
+      data,
+      releaseMetdataConfig.releaseNameColumnName,
+    ),
+    releaseEntity: getValueFromData(
+      schema,
+      data,
+      releaseMetdataConfig.releaseEntityColumnName,
+    ),
+    releaseDate: formatReleaseDate(
+      schema,
+      data,
+      releaseMetdataConfig.releaseDateColumnName,
+    ),
```

Comment:
Lookup release metadata in the source table, using the column names provided in the config